process-engine/bpmn-studio
# 🐛 Fix Clearing TokenViewer

## Changes

1.  Fix Clearing TokenViewer
2. Use 'ClearTokenViewer' to Clear the TokenViewer

PR: #1942

## How to test the changes

- Open the Inspect View
- Open the TokenViewer
- Select any FlowNode containing a token
- Select any diagram from a local solution
- **Notice that the TokenViewer still displays the token from the previously selected FlowNode.**
